src/components/forms/components/submission-fields.js

```diff
@@ -489,12 +490,24 @@ class ArrayField extends React.Component{
 
     componentDidUpdate(prevProps, prevState){ // We can't do a comparison of props.value here because parent property mutates yet stays part of same obj.
         const { value, field, pushArrayValue, modifyNewContext, nestedField, schema, linkType } = this.props;
+        const { maxItems } = schema;
+
         if (ArrayField.shouldPushArrayValue(value, field)){
+            if (maxItems && value.length == maxItems)
```

Comment:
It is considered bad practice to use `==` instead of `===` in JavaScript. This is because `==` in JavaScript doesn't check type and things like `false == 0` and `false == ""` will evaluate to true, and this is often a source of hard-to-trace bugs. [More Info.](https://bonsaiden.github.io/JavaScript-Garden/#types.equality)

If know for a fact that are comparing two variables of the same type, then `==` is safe to use, but there's no real reason to since `===` is about as performant and also type-safe.